Sources/HipMobileUI/Location/NearbyExhibitManager.cs

```diff
@@ -82,10 +80,19 @@ public class NearbyExhibitManager : INearbyExhibitManager
                         }
                     }
 
-                    NavigationViewModel nv = new ExhibitPreviewViewModel(e, this);
-                    await
-                        IoCManager.Resolve<INavigationService>()
-                                  .PushModalAsync(nv);
+                    // Display popup page or local notification
+                    if (appMinimized)
+                    {
+                        var notificationPlayer = IoCManager.Resolve<INotificationPlayer>();
+                        notificationPlayer.DisplayExhibitNearbyNotification(e.Name, "Besuche eine nahe Sehenswürdigkeit!", e.Image.Data);
+                    }
+                    else
+                    {
+                        NavigationViewModel nv = new ExhibitPreviewViewModel(e, this);
```

Comment:
Why not use `var` here?

Reply:
Just reused the existing code. Will change it.